tests/sqlparser_mssql.rs

```diff
@@ -2053,3 +2054,171 @@ fn parse_drop_trigger() {
         }
     );
 }
+
+#[test]
+fn parse_mssql_go_keyword() {
+    let single_go_keyword = "USE some_database;\nGO";
```

Comment:
Can we have one of the test cases use `ms().one_statement_parses_to(sql, canonical)`? looking at the tests its unclear what the behavior of the parser is in terms of serializing an AST that has GO statements in it, and that it needs special behavior to inject new lines manually I imagine

Reply:
I'm unsure the best way to approach this question. `one_statement_parses_to` requires _one_ statement, right? But most of these SQL text examples are multiple statements. I didn't see anything in TestUtils for a series of statements, so I added a new helper accordingly with canonical strings, let me know if that covers it

Reply:
Ah yeah a new helper sounds reasonable to me thanks!